pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -820,16 +833,24 @@ func (o *MustGatherOptions) newPod(node, image string) *corev1.Pod {
 
 // BackupGathering is called if the full must-gather has an error.  This is useful for making sure we get *something*
 // no matter what has failed.  It should be focused on universal openshift failures.
-func (o *MustGatherOptions) BackupGathering(ctx context.Context, errs []error) {
+func (o *MustGatherOptions) BackupGathering(ctx context.Context, namespace string, errs []error) {
+	targets := []string{
+		"clusterversion.v1.config.openshift.io",
```

Comment:
I'm also adding ClusterVersion to the fallback inspection, because it's often useful to know what the cluster-version operator is up to.  But I'm happy to spin that out into a separate pull request  if folks would prefer distinct pivots.

Reply:
It would be better to do that in separate PR.